src/main/webapp/assets/js/foodMeetings.js

```diff
@@ -94,24 +85,60 @@ $(document).ready(function () {
 		}
 	});
 
-	$("#ImageLinkId").change(function (event) {
-
-		var imageLink = $("#ImageLinkId").val();
-
-		$("#ImageCard").attr("src",imageLink);
-	});
-
 	$("#AddNewMeeting").click(function (event) {
 		
```

Comment:
I think those spaces at the beginnig and at the end of a function are unecessary.